src/probabilities.jl

```diff
@@ -178,11 +181,23 @@ Works for estimators that implement [`total_outcomes`](@ref).
 
 See also: [`MissingDispersionPatterns`](@ref).
 """
-function missing_outcomes end
-
 function missing_outcomes(x::Array_or_Dataset, est::ProbabilitiesEstimator)
     probs = probabilities(x, est)
     L = total_outcomes(x, est)
     O = count(!iszero, probs)
     return L - O
 end
+
+"""
+    all_possible_outcomes([x,] est::ProbabilitiesEstimator) → o
+
+Return a vector containing all possible outcomes of `est`.
+Only possible for estimators that implement [`total_outcomes`](@ref),
```

Comment:
This is an incomplete sentence. I propose "Getting/obtaining the outcome space is only possible for ...." instead.